Default.sublime-keymap

```diff
@@ -471,13 +471,20 @@
         ]
     },
     {
-        "keys": ["f"],
+        "keys": ["f", "d"],
```

Comment:
Yeah, I was thinking "diff history" as the short-description.  Generally, unless grouping very-related commands together (like stash commands), I try to stick to single key-presses.  This new command relates to the diff command in that it sort of does something similar.  Sticking to single key-bindings (when possible) also avoids the issue of mistakenly invoking the wrong command when the first key wasn't registered.